scripts/release.sh

```diff
@@ -2,32 +2,56 @@
 
 COLOR_RED='\033[0;31m'
 COLOR_NONE='\033[0m'
+
+DEFAULT_BRANCH='main'
 CURRENT_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 
 if [ $CURRENT_GIT_BRANCH != 'main' ]; then
   printf "\n"
-  printf "${COLOR_RED} Error: The release.sh script must be run while on the main branch. \n ${COLOR_NONE}"
+  printf "${COLOR_RED} Error: Must be on main branch to create a new release. \n ${COLOR_NONE}"
   printf "\n"
 
   exit 1
 fi
 
-if [ $# -ne 1 ]; then
-  printf "\n"
-  printf "${COLOR_RED} Error: Release version argument required. \n\n ${COLOR_NONE}"
-  printf " Example: \n\n    ./tools/release.sh 0.9.0 \n\n"
-  printf "  Example (make): \n\n    make release version=0.9.0 \n"
-  printf "\n"
+SRCDIR=${SRCDIR:-"."}
+GOBIN=$(go env GOPATH)/bin
+VER_PACKAGE="internal/version"
+VER_CMD=${GOBIN}/svu
+VER_BUMP=${GOBIN}/gobump
+CHANGELOG_CMD=${GOBIN}/git-chglog
+CHANGELOG_FILE=CHANGELOG.md
+SPELL_CMD=${GOBIN}/misspell
+
+# Compare versions
+VER_CURR=$(${VER_CMD} current --strip-prefix)
+VER_NEXT=$(${VER_CMD} next --strip-prefix)
+
+echo " "
+echo "Comparing tag versions..."
+echo "Current version: ${VER_CURR}"
```

Comment:
@sanderblue do you have an example of what VER_CURR would look like? I believe we want something like vX.Y.Z (notice the v prefix).

Reply:
Example value will be: `0.0.1` 

It will be without the `v` prefix because the variable is set using `--strip-prefix` 

i.e. `svu current --strip-prefix`

We actually may not need to strip the prefix here though. We needed to in the client because we were injecting the value into the client's `version.go` file in the release script, but that's not how the CLI handles this due it being a binary vs a library.

Reply:
Yep, that was the reason. I can update it so we don't strip the prefix in this script.

Reply:
Updated